SimDataFormats/GeneratorProducts/interface/LHEEventProduct.h

```diff
@@ -24,6 +24,27 @@ class LHEEventProduct {
 	LHEEventProduct(const lhef::HEPEUP &hepeup,
 			const double originalXWGTUP) : 
 	  hepeup_(hepeup), originalXWGTUP_(originalXWGTUP) {}
+	LHEEventProduct(LHEEventProduct&& other) {
+	  hepeup_ = std::move(other.hepeup_);
+	  comments_ = std::move(other.comments_);
+	  pdf_ = other.pdf_; // auto_ptr, so copy is actually a move
+	  weights_ = std::move(other.weights_);
+	  originalXWGTUP_ = std::move(originalXWGTUP_);
+	  scales_ = std::move(scales_);
+	  npLO_ = std::move(npLO_);
+	  npNLO_ = std::move(npNLO_);
+	}
+	LHEEventProduct& operator=(LHEEventProduct&& other) {
+	  hepeup_ = std::move(other.hepeup_);
+	  comments_ = std::move(other.comments_);
+	  pdf_ = other.pdf_; // auto_ptr, so copy is actually a move
+	  weights_ = std::move(other.weights_);
+	  originalXWGTUP_ = std::move(originalXWGTUP_);
+	  scales_ = std::move(scales_);
+	  npLO_ = std::move(npLO_);
+	  npNLO_ = std::move(npNLO_);
```

Comment:
Hi @makortel , I'm MC generator validator of PdmV group. While validating 10_1_0_pre2, I've found that some informations such as npLO & npNLO are missing in generated MC samples. It seems that a bug is introduced from this PR, especially SimDataFormats package. I tried adding `other`. in L32-35 & L42-45, but simply adding it could not fix the issue. May I ask you to review this PR? Any questions would be appreciated.

Reply:
@SanghyunKo Whoops, that's certainly a bug (sorry about that). But I'm puzzled why adding `other.` on the right-hand sides of lines 32-35 and 42-45 would not fix the issue. Could you give me a (simple) recipe how to check myself whether  the bug is fixed or not?

Reply:
@SanghyunKo 
I see the `other` was already added by #22429 (merged in 10_1_0_pre3). I'm nevertheless interested to double-check if the fix actually works or not, so 

> Could you give me a (simple) recipe how to check myself whether the bug is fixed or not?

Reply:
@makortel Here I attached link to dropbox that contains simple command that I used to test it, python configuration file and some example of generated LHE file in pre1 & pre2 respectively. I'm also puzzled but let's cross-check it first.

[Link to dropbox](https://www.dropbox.com/s/wvwsmlqw9tvzmha/npLOandnpNLO_v2.zip?dl=0)

Reply:
@SanghyunKo 
Thanks for letting me know you got it to work as well (I can also confirm it with the test you provided).